website/blog/2020-11-06-sbt-BSP-support.md

```diff
@@ -0,0 +1,147 @@
+---
+author: Chris Kipp
+title: sbt BSP support
+authorURL: https://twitter.com/ckipp01
+authorImageURL: https://avatars1.githubusercontent.com/u/13974112?s=400&u=7b6a2ddab8eec6f99e4e40ae9b81f71cb5ba92e5&v=4
+---
+
+If you've been following the [sbt
+releases](https://github.com/sbt/sbt/releases), 1.4.x introduced some great new
+features, one of those being BSP support. This effort was initiated by community
+members and proposed to the [Scala Center Advisory
+Board.](https://github.com/scalacenter/advisoryboard/blob/master/proposals/023-bsp.md).
+Then thanks to the work of [Adrien Piquerez](https://twitter.com/adrienpi2) and
+[Eugene Yokota ](https://twitter.com/eed3si9n) BSP support became a reality in
+1.4.0. You can read a more detailed account of the reason behind the work and
+some of the details of the implementation in this blog post, [BSP Support in sbt
+1.4](https://www.scala-lang.org/blog/2020/10/27/bsp-in-sbt.html).
+
+The Metals team is also happy to announce that as of 0.9.5, the process to use
```

Comment:
Ah, I was sort of thinking they would be so close together it didn't matter, but no worries. I changed it to `upcoming 0.9.5 release`

Reply:
Actually thinking about this a bit more, these instructions mention the new `generate-bsp-config` which users won't have until the new release of the VS Code extension, so maybe we shouldn't advertise this too much before the actual release to avoid confusion? wdyt?

Reply:
The release notes will come up only after the VS Code extension is also released, so it should be fine.

Reply:
We can also try to maybe publish it at the same time then? No need for the upcoming in that case.

Reply:
Sounds good 👍